### PR TITLE
Update to fastify@4

### PR DESCRIPTION
**Pull request by mcollina**
Signed-off-by: Matteo Collina <hello@matteocollina.com>

#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Review by climba03003 — APPROVED** [review]
LGTM.

File: index.js [hunk 1]
```diff
@@ -338,7 +338,7 @@ async function fastifyStatic (fastify, opts) {
           file = file
             .replace(rootPath.replace(/\\/g, '/'), '')
             .replace(/^\//, '')
-          const route = encodeURI(prefix + file).replace(/\/\//g, '/')
+          const route = (prefix + file).replace(/\/\//g, '/')
```
Comment (Fdawgs) [inline index.js:341]: Not needed anymore?

Comment (climba03003) [reply index.js:341]: You can follow the discussion here
https://github.com/delvedor/find-my-way/pull/253#issuecomment-1118516930

Comment (Fdawgs) [reply index.js:341]: Ah brill, thanks @climba03003

**Pull request merged**